cmd/minikube/cmd/start.go

```diff
@@ -113,6 +115,8 @@ func runStart(cmd *cobra.Command, args []string) {
 	libMachineClient := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
 	defer libMachineClient.Close()
 
+	setProxy()
```

Comment:
@hferentschik we can add proxy to cluster.MachineConfig struct but to pass it till docker daemon we should have such option in the docker-machine side also https://github.com/minishift/minishift/blob/master/pkg/minikube/cluster/cluster.go#L174 here engine.Options doesn't have proxy argument but it is only supported by dockerEnv which take care of daemon options.